.github/workflows/release.yml

```diff
@@ -7,29 +7,14 @@ on:
 jobs:
   cockpituous:
     runs-on: ubuntu-latest
-    environment: release
     container:
```

Comment:
Explicit `permissions:` line for the `GITHUB_TOKEN` please (uploading the release I guess?)

Reply:
Done as a separate commit. I found https://docs.github.com/en/rest/overview/permissions-required-for-github-apps which is what I had been looking for for a long time, to document what these permissions do exactly.